### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by ALegchilov**

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"use strict";
```
Comment (AlexGalichenko) [inline lib/cli.js:1]: to gitignore

File: lib/crypt.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+"use strict";
```
Comment (AlexGalichenko) [inline lib/crypt.js:1]: to gitignore

File: lib/index.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+"use strict";
```
Comment (AlexGalichenko) [inline lib/index.js:1]: gitignore

File: tsconfig.json [hunk 4]
```diff
@@ -2,9 +2,6 @@
   "include": [
     "src/**/*.ts"
   ],
-  "exclude": [
```
Comment (AlexGalichenko) [inline tsconfig.json:5]: return this to tsconfig

**Review by AlexGalichenko — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 4 above)
Comment (ALegchilov) [reply tsconfig.json:5]: This code does nothing as only content of `src` folder included

File: lib/cli.js (hunk 1 above)
Comment (ALegchilov) [reply lib/cli.js:1]: Why the content of the `lib` folder is the application itself

**Review by KirillBogdanets — APPROVED** [review]

**Pull request merged**